modules/kuberay-cluster/kuberay.tf

```diff
@@ -25,19 +25,26 @@ resource "helm_release" "ray-cluster" {
   namespace        = var.namespace
   create_namespace = var.create_namespace
   version          = "0.6.1"
-  values = var.enable_autopilot ? [templatefile("${path.module}/kuberay-autopilot-values.yaml", {
-    gcs_bucket          = var.gcs_bucket
-    k8s_service_account = var.google_service_account
-    grafana_host        = var.grafana_host
-    })] : (var.enable_tpu ? [templatefile("${path.module}/kuberay-tpu-values.yaml", {
+
+  values = [
```

Comment:
I wonder if there is a better way to do this templating - this could get ugly  (already is ugly)